content/browser/frame_host/cross_process_frame_connector.h

```diff
@@ -153,6 +156,10 @@ class CONTENT_EXPORT CrossProcessFrameConnector
   bool is_throttled_ = false;
   bool subtree_throttled_ = false;
 
+  // Says whether |this| set a new view, and waits before the |view_| receives a
```

Comment:
nit: ... |this| is in the process of setting  a new view, and waiting...?

Reply:
done